.gitignore

```diff
@@ -11,6 +11,9 @@ lib-cov
 pids
 logs
 results
+_notes/
```

Comment:
This will be done via separate PR - protects against Dreamweaver _notes files are created